services/release-controller-test/admin_02_rbac.yaml

```diff
@@ -0,0 +1,183 @@
+kind: List
+apiVersion: v1
+items:
+- kind: ServiceAccount
+  apiVersion: v1
+  metadata:
+    name: release-controller-test
```

Comment:
Not sure to what extent we care but `stg` or `stage` might be nicer than `test` if we intend for this to always be running some unpublished version for validation